spec/factories/category.rb

```diff
@@ -1,7 +1,21 @@
 FactoryGirl.define do
   factory :category, class: Category do
-    name        "main course"
-    description "for main course items"
+    name          "main course"
+    description   "for main course items"
   end
-  
-end
\ No newline at end of file
+
+  factory :category1, class: Category do
+    name          "drinks"
+    description   "different kinds"
+  end
+
+  factory :category2, class: Category do
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.